src/Commands/PurgeCommand.php

```diff
@@ -169,7 +169,7 @@ protected function purgeExpiredRecordsForModel($model, $days)
             return false;
         }
 
-        if (!method_exists($model, 'onlyTrashed') || !method_exists($model, 'forceDelete')) {
```

Comment:
@ssfinney Is this possible now that we are targeting later versions of Laravel?

Reply:
@jimmypuckett Even if so, why try to check for both methods at all?

Reply:
I am just making sure that this is the right thing to do with the discussion that we had in #21.

There is one of those methods that is for one version of Laravel & then another for another version.

I am just asking if we ware doing the right one now that we are dropping old versions

Reply:
Oh ok, I understand now @jimmypuckett.

Yes we are. As of L >=5.5 the `onlyTrashed` method is gone, and the `forceDelete` is the right one to check.